### ISSUE TITLE
Option --overwrite does not overwrite database folder created by --db-cluster

### ISSUE THREAD

**Issue by vovikhangcdv** [github/codeql-cli-binaries#156]
## Description
When creating database by using `--db-cluster`, it can not be overwritten by option `--overwrite`.

## Reproduce
1. Create a database of ***a multi-language source*** (use auto-detect languages)
```
codeql database create --db-cluster -- test-db  
```
2. Overwrite the database
```
codeql database create --overwrite --db-cluster -- test-db  
```

## Error message
```
A fatal error occurred: Refusing to create databases: '[]/test-db' exists and is not an empty directory.
Please specify a different location, or use the --overwrite flag to overwrite existing CodeQL databases.
```

**Comment by aeisenberg** [comment]
Thanks for reporting. I can reproduce this. It does appear to be a bug.

**Comment by aeisenberg** [comment]
A fix will be available in codeql v2.12.2.

**Comment by vovikhangcdv** [comment]
thanks @aeisenberg!
